templates/repo/issue/list.tmpl

```diff
@@ -171,6 +171,7 @@
 					{{else}}
 						<div class="ui red active basic button issue-action" data-action="close" data-url="{{$.RepoLink}}/issues/status" style="margin-left: auto">{{.locale.Tr "repo.issues.action_close"}}</div>
 					{{end}}
+					<div class="ui negative active button issue-action" data-action="delete" data-url="{{$.RepoLink}}/issues/status" style="margin-left: auto">{{.locale.Tr "repo.issues.delete"}}</div>
```

Comment:
I'm strongly against allowing such a destructive action to occur without presenting a warning dialogue.
At best, extract the existing issue deletion dialogue into a new template and reuse it, or copy it if that's not easily possible.

Reply:
In addition, why display a text when it's probably a better idea to use the `octicon-trash` SVG instead?
First of all to minimize the user potential to missclick there, and secondly to improve the UX.